modules/elasticsearch_helper_index_alias/elasticsearch_helper_index_alias.routing.yml

```diff
@@ -0,0 +1,39 @@
+elasticsearch_helper_index_alias.manage_alias_controller.indices_status:
+  path: '/elasticsearch_helper/index_alias/indices'
```

Comment:
All of these paths should use the same structure as the main module `/admin/config/search/elasticsearch_helper`